util/find-doc-nits

```diff
@@ -862,6 +889,12 @@ sub loadnum ($;$) {
         next if /^#/;
         next if /\bNOEXIST\b/;
         my @fields = split();
+        if ($type && ($type eq "crypto" || $type eq "ssl")) {
+            # 3rd field is version
+            if (not $fields[2] eq "3_0_0") {
+                $history{$fields[0].'(3)'} = $type.$fields[2];
```

Comment:
This will be broken once we switch to 4.0.0. But I assume we can fix that once we do that.

Reply:
Why? Is the idea to not document new functions arriving in 4.0 as having done that (then)?

Reply:
Ouch. Interesting to know. So basically all functions still not having a HISTORY entry at that time would get flagged. But whether that's an issue can indeed be decided then. This PR will at least ensure it's only functions added before or at 3.0.0, right?